server/routes/game.js

```diff
@@ -0,0 +1,94 @@
+const router = require('express').Router();
+const {GameTable, PlayerSession} = require('../db');
+
+//this post request creates a new game room
+router.post('/', async (req, res, next) => {
+    try {
+        let accessCode;
+        let created = false;
+        let foundTable;
+
+        while (!created) {
+            //radix 36 uses 0-9 and a-z
+            accessCode = Math.random().toString(36).substr(2, 6).toUpperCase()
+
+            //check if table with this access code exists
+            foundTable = await GameTable.findOne({where: {accessCode}})
+
+            //if this access code hasn't been used, use it. Otherwise, loop around and try a new code.
+            if (!foundTable) {
+                const newGame = await GameTable.create({accessCode})
+                res.send(newGame)
+                created = true
+            }
+        }
+    }
+    catch(err) {
+        next(err)
+    }
+})
+
+//this get request will place someone in a room (if allowed) according to their cookie and db
+//206 if the table was full
+//204 if the table doesn't exist
+router.get('/:gameId', async (req, res, next) => {
+    try {
+
+        const gameTable = await GameTable.findOne({
+            where: {
+                accessCode: req.params.gameId
+            }
+        })
+
+        const player = await PlayerSession.findOne({
+            where: {
+                id: req.cookies.sid
+            }
+        })
```

Comment:
Not a huge issue and probably not worth refactoring but since player and gametable don't depend on each other you can probably promise.all these